torchsynth/module.py

```diff
@@ -600,14 +512,14 @@ def _forward(self, control_in: Signal, audio_in: Signal) -> Signal:
         return control_in * audio_in
 
 
-class TorchNoise(TorchSynthModule1D):
+class TorchNoise(TorchSynthModule):
     """
     Adds noise to a signal
 
     Parameters
     ----------
     ratio (float): mix ratio between the incoming signal and the produced noise
-    **kwargs: see TorchSynthModule
+    **kwargs: see TorchSynthModule0Ddeprecated
```

Comment:
Same as above

Reply:
mechanical error. I fixed in later PR but push here anyway.